.github/workflows/build_deploy.yml

```diff
@@ -20,34 +20,134 @@ concurrency:
   cancel-in-progress: false
 
 jobs:
-  build:
+  #
+  # For each document category (admin, standards, etc.),
+  # build using `metanorma site generate`.
+  #
+  # At the very end, integrate them all with Jekyll.
+  #
+  # NOTE: Try to parallelize the build process as much as possible.
+  # This is currently done by building the admin and standards docs in
+  # parallel.
+  #
+  build-admin:
+    name: Build admin docs
+    needs: build-site
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: 'recursive'
+          depth: 1
+      - name: Debug environment
+        run: |
+          ruby -e 'pp ENV'
+      - name: Use Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+      - name: Build site
+        run: |
+          make build-admin
+      - name: Save artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: admin-docs
+          path: _site/administrative
+
+  build-standards:
+    name: Build standards docs
+    needs: build-site
```

Comment:
Is the approach now to build 3 sites independently and then combining them in the directory to publish? Are we able to use the CalConnect styling for the metanorma mini-sites?

Reply:
> Is the approach now to build 3 sites independently and then combining them in the directory to publish?

Yes.

> Are we able to use the CalConnect styling for the metanorma mini-sites?

Not at the moment.  Will investigate.

Reply:
It seems this approach requires a set of `.liquid` templates (via relaton, via metanorma-cli) for the CalConnect style.  If there exists none, I'll create them...

Reply:
Observation: `{% include ` tag as used in Jekyll is not available in Relaton, as it is specific to Jekyll: https://github.com/jekyll/jekyll/blob/master/lib/jekyll/tags/include.rb

Meanwhile, Relaton uses [straight `liquid`](https://shopify.github.io/liquid/tags/template/).

This makes the current set of Jekyll `_includes` and `_layouts` unreusable.

Reply:
Just in case you are not aware of the original approach, which is to build the documents, and input the Relaton Bib YAML files into Jekyll to render the site. This way Jekyll reads the Relaton files as native "data files".